Cargo.toml

```diff
@@ -48,11 +48,11 @@ broadcaster = { version = "0.2.6", optional = true, default-features = false, fe
 femme = "1.2.0"
 surf = "1.0.2"
 tempdir = "0.3.7"
+futures-preview = { version = "=0.3.0-alpha.19", features = ["async-await"] }
```

Comment:
We can also drop the `async-await` features, we don't need it anymore.

Reply:
I think examples/a-chat is still using it.
https://github.com/async-rs/async-std/blob/5881d9addef991e6d261a073f81c961ee3eb4bad/examples/a-chat/client.rs#L1

Reply:
Right, in this case, we should keep it.